WSRequestHandler.h

```diff
@@ -63,6 +63,7 @@ class WSRequestHandler : public QObject {
     static void HandleSetSceneItemPosition(WSRequestHandler* req);
     static void HandleSetSceneItemTransform(WSRequestHandler* req);
     static void HandleSetSceneItemCrop(WSRequestHandler* req);
+    // Add handler here
```

Comment:
This is just a header file so all it really needs is the method name.
`static void HandleGetSceneItemSceneProperties(WSRequestHandler* req);` or whatever you settle on.